.gitignore

```diff
@@ -29,7 +29,7 @@ vendor/
 license.key
 
 # generated from merging multiple files in the Makefile
-config/all-in-one-flavor-*.yaml
+config/all-in-one.yaml
```

Comment:
Even though this is generated do you think it might be useful to keep it checked in? Since it is not just cating the files together but also running kustomize on them. As is the CRDs that are in the repo are not actually useful since you really only want to apply the all in one